toolchest_client/files/unpack.py

```diff
@@ -34,10 +33,6 @@ def unpack_files(file_path_to_unpack, is_compressed):
         unpacked_paths = ["/".join([unpacked_outputs_dir, file_name]) for file_name in unpacked_file_names]
         unpacked_file_paths = [os.path.normpath(path) for path in unpacked_paths if os.path.isfile(path)]
 
-        # If only 1 file is unpacked, just return path instead of [path].
-        # This is to be consistent with the return value from the other output types.
-        if len(unpacked_file_paths) == 1:
-            return unpacked_file_paths[0]
```

Comment:
This is an interface change: when a `.tar.gz` is returned, `output_file_paths` was previously:
- a list if there were multiple files present
- a single string if there was only one file present

That's suboptimal – imagine a situation in which you want to `[] += output.output_file_paths`. A string would return `['f', 'i', 'l', 'e']`, and a list would return `['file']`.

I hit this issue on the infra-side, so adjusted it in this PR

Reply:
I don't think this is a breaking change, because I couldn't find any tools that are compressing output which don't have multiple files at this point